pkg/i2gw/providers/kong/converter_test.go

```diff
@@ -33,7 +33,9 @@ import (
 
 func Test_ToGateway(t *testing.T) {
 	iPrefix := networkingv1.PathTypePrefix
+	isPathType := networkingv1.PathTypeImplementationSpecific
```

Comment:
isPathType is confusing as I expect it to be a `bool` while I think you refer to implementationSpecificPathType
maybe `implementationSpecificPathType` or `implSpecificPathType` ?

Reply:
Yeah, I agree 😄 Renamed